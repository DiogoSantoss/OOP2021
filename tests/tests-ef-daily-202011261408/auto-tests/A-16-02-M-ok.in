8
2
R2
1
B1937
1
3
P1
1C6H5OH
10
s
8B1256
20
n
2
S2
1
8B1256
4
3
P2
7M5
30
n
1
0
1
1
1
2
1
3
0
0
