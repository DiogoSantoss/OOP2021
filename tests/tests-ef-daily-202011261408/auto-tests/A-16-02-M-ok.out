Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do fornecedor: Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do fornecedor: Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 0|R2|B1937|1|5|5|1
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 1|P1|1180|0
1C6H5OH|10
8B1256|20
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 2|S2|8B1256|4|232|232|1
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 3|P2|240|0
7M5|30
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 